spyderlib/plugins/shortcuts.py

```diff
@@ -159,17 +63,12 @@ def data(self, index, role=Qt.DisplayRole):
         column = index.column()
         if role == Qt.DisplayRole:
             if column == CONTEXT:
-                return to_qvariant(shortcut.context)
+                return to_qvariant(_(shortcut.context.capitalize()))

```

Comment:
I don't know if this is enough to get things localized.

Don't we need to pass a real string to `_(...)` instead of something computed from a method/function, so that `gettext` grabs that string for translation?

I say it because I haven't seen any example in our sourcecode of `_(...)` applied to something that's not a string :-)